script.js

```diff
@@ -0,0 +1,13 @@
+//hamburger menu
+const hamburger = document.querySelector('.hamburger');
+const mobileMenu = document.querySelector('.mobile-menu');
+
+hamburger.addEventListener('click', () => {
+hamburger.classList.toggle('active');
+mobileMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.menu-item').forEach((n) => n.addEventListener('click', () => {
+hamburger.classList.remove('active');
+mobileMenu.classList.remove('active');
```

Comment:
- [ ] Kindly make sure to use backdrop for the background so that background text is displayed as per the design.
- [ ] Kindly make sure to follow the spacing for the mobile menu options.

https://user-images.githubusercontent.com/70269507/173334368-ea93609a-2d84-453c-af68-2ad3f3a99b59.mov